docs/api.md

```diff
@@ -1961,6 +1962,11 @@ await aHandle.dispose();
 await resultHandle.dispose();
 ```
 
+#### executionContext.frame()
+- returns: <?[Frame]> Frame associated with this execution context.
+
+> **NOTE** Not every execution context is associated with a frame.
```

Comment:
It looks like workers and content scripts have null frames? The note here is not helpful, it doesn’t explain why.

Reply:
@JoelEinbinder added a better explanation.